powerflex/sdc_datasource_test.go

```diff
@@ -43,12 +43,12 @@ func TestSdcDataSource(t *testing.T) {
 				Config: ProviderConfigForTesting + TestSdcDataSourceByEmptyBlock,
 				Check: resource.ComposeAggregateTestCheckFunc(
 					resource.TestCheckTypeSetElemNestedAttrs("data.powerflex_sdc.selected", "sdcs.*", map[string]string{
-						"id":   "e3ce1fb500000000",
-						"name": "Terraform_sdc",
+						"sdcs.0.id":   "e3ce1fb500000000",
+						"sdcs.0.name": "terraform_sdc",
```

Comment:
This does not seem correct, and the build is failing at this step too.
I think this may be correct
```suggestion
						"sdcs.id":   "e3ce1fb500000000",
						"sdcs.name": "terraform_sdc",
```